src/test/java/com/github/dockerjava/api/model/BindTest.java

```diff
@@ -62,4 +98,19 @@ public void toStringDefaultAccessMode() {
         assertEquals(Bind.parse("/host:/container").toString(), "/host:/container:rw");
     }
 
+    @Test
+    public void toStringReadOnlySEL() {
+        assertEquals(Bind.parse("/host:/container:ro,Z").toString(), "/host:/container:ro,Z");
```

Comment:
Plz elaborate on techical neccessity of doing so. It looks like pure syntactical change.